src/main.rs

```diff
@@ -44,7 +44,7 @@ fn main() {
     match arg.as_str() {
       "-v" | "--verbose" => verbose = true,
       _ => {
-        if expression_opt == None {
+        if expression_opt.is_none() {
```

Comment:
Don't really see the benefit of this one

Reply:
Here's the clippy lint for it:
https://rust-lang.github.io/rust-clippy/master/index.html#partialeq_to_none

and a more detailed discussion here:
https://github.com/rust-lang/rust-clippy/issues/9275

Reply:
That does seem fair, thanks for the links!